lldb/source/Target/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -280,6 +280,20 @@ class LLDBMemoryReader : public swift::remote::MemoryReader {
   bool queryDataLayout(DataLayoutQueryType type, void *inBuffer,
                        void *outBuffer) override {
     switch (type) {
+    // FIXME: add support for case DLQ_GetPtrAuthMask:
+    case DLQ_GetObjCReservedLowBits: {
```

Comment:
Nice I was wondering what to do about this, but I figured it's good that the warning is there to tell us that it's a todo.